source/administration/devops-command-center.rst

```diff
@@ -116,9 +194,9 @@ To open the Incidents backstage, which lists all incidents associated with the c
 Viewing incident details
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-To view details of active incidents, select **! Incidents** from the plugin menu, or **!** in the channel header to open the RHS panel.
+To view details of active incidents associated with your Mattermost team, select **!** in the channel header to open the RHS panel. If there are no active incidents, you can view details of incidents that have ended. If an incident is public you can view details. If an incident is private you will not be able to view the details.
```

Comment:
There is no longer a list of incidents in the RHS. The **!** icon just shows the details of the incident if in an incident channel. If not, it suggests to create a new one.